src/Uno.UI/Media/PathMarkupParser.cs

```diff
@@ -566,8 +566,11 @@ private double ReadDouble(ref ReadOnlySpan<char> span)
 			{
 				throw new InvalidDataException("Invalid double value");
 			}
-
+#if NET7_0_OR_GREATER
```

Comment:
This source file is shared with the source generators project. That is why this conditional is needed.